scalafmt-tests/src/test/resources/align/ArrowEnumeratorGenerator.stat

```diff
@@ -13,4 +14,63 @@ for {
       }
   y <- Int(2)
 } yield x + y
-
+<<< split long lines
+for {
+  variable <-
+    record.field.field1.map(_.toString)
+  cond <-
+    if (variable) doSomething
+    else doAnything
+} yield cond
+>>>
+for {
+  variable <- record.field.field1
+               .map(_.toString)
+  cond <- if (variable) doSomething
+         else doAnything
+} yield cond
+<<< insert newlines to for if no newlines origin
+for {
+  variable <- record.field.field1.map(_.toString)
+  cond <- if (variable) doSomething else doAnything
+} yield cond
+>>>
+for {
```

Comment:
It's really strange formatting

```scala
  cond <- if (variable) doSomething
         else doAnything
```

Is it by design of `align.arrowEnumeratorGenerator = true`?

(It is not new behavior from this PR, formatting on v2.4.2 is same)

Reply:
agreed. i think someone complained on gitter about this.

Reply:
created issue https://github.com/scalameta/scalafmt/issues/1797